toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -73,59 +68,23 @@ func buildDracutModule(dracutModuleName string, imageChroot *safechroot.Chroot)
 
 	// Check if the dracut module configuration file already exists.
 	if _, err := os.Stat(dracutConfigFile); os.IsNotExist(err) {
-		lines := []string{"add_dracutmodules+=\"" + dracutModuleName + "\""}
+		lines := []string{
+			"add_dracutmodules+=\"" + dracutModuleName + "\"",
+			"add_drivers+=\"" + dracutDriverName + "\"",
+		}
 		err = file.WriteLines(lines, dracutConfigFile)
 		if err != nil {
 			return fmt.Errorf("failed to write to dracut module config file (%s): %w", dracutConfigFile, err)
 		}
 	}
 
 	err = imageChroot.Run(func() error {
-		err = shell.ExecuteLiveWithErr(1, "dracut", "-f", "--kver", kernelVersion)
+		initrdImagePath := "/boot/initrd.img-" + kernelVersion
+		err = shell.ExecuteLiveWithErr(1, "mkinitrd", "-f", initrdImagePath, kernelVersion)
```

Comment:
I missed the conversation on this one, but why are we switching frim dracut to mkinitrd?

Reply:
Added to the description, please check.